### ISSUE TITLE
Trouble getting started on MacOS

### ISSUE THREAD

**Issue by mrax1** [unpoller/unpoller#369]
Complete newbie. Got it set up through MacOs terminal but installing influxDB, Grafana, and Unpoller. 
When I run "brew services list" in Home Brew I get an error on unpoller. Not sure why. 

Name     Status  User       Plist
grafana  started mtock /Users/mwickham25/Library/LaunchAgents/homebrew.mxcl.grafana.plist
influxdb started mtock /Users/mwickham25/Library/LaunchAgents/homebrew.mxcl.influxdb.plist
unpoller error    mtock /usr/local/opt/unpoller/homebrew.mxcl.unpoller.plist

I am not sure if this is a problem. I am also not sure how to start the actual program. 

I am also not sure if I created the proper config file. I started nano and put the following in based on the UnPoller website. 
[unifi]
  url = "http://192.168.1.1”
  user = “mtock”
  pass = “mtock123"
  save_sites = true
  save_ids = false
  save_events = false
  save_alarms = false
  save_dpi = false
  sites = [ "default" ]

I saved this file to "/usr/local/etc/unpoller/up.conf"

Any help would be great, thank you.

**Comment by davidnewhall** [comment]
> When I run "brew services list" in Home Brew I get an error on unpoller. Not sure why.

That sounds like the problem. Please share.

**Comment by davidnewhall** [comment]
The log file should be in `/usr/local/log`, check that too.

**Comment by mrax1** [comment]
How do I get to this file? Just open it up and post the contents here?

**Comment by davidnewhall** [comment]
What happens when you just run the app? Type `/usr/local/bin/unifi-poller`

**Comment by mrax1** [comment]
This is what I get when I run that.  Should the command have unifi-poller in it even though the name has changed to unpoller?

2021/08/18 14:40:48 [ERROR] finding file: stat /usr/local/etc/unifi-poller/up.conf: no such file or directory

**Comment by davidnewhall** [comment]
Unfortunately the repo has been renamed but the app has not been renamed yet. Everything still uses `unifi-poller`. Move your config file to  `/usr/local/etc/unifi-poller/up.conf`.

**Comment by mrax1** [comment]
Wow thank you!!!  It was pointing to /usr/local/etc/unpoller/up.conf.  Of course that was not the right directory. Now when I run
/usr/local/etc/unifi-poller/up.conf the program starts up and this is what I get:

Program loads the config file but seems to hang up on something with the influxdb. 

mw25@W-MBP ~ % /usr/local/bin/unifi-poller
2021/08/18 15:21:39 [INFO] Loading Configuration File: /usr/local/etc/unifi-poller/up.conf
2021/08/18 15:21:39 [INFO] UniFi Poller v2.1.3 Starting Up! PID: 17520
2021/08/18 15:21:58 [INFO] Found 1 site(s) on controller https://192.168.1.1: default (Default)
2021/08/18 15:21:58 [INFO] Configured UniFi Controller 1 of 1:
2021/08/18 15:21:58 [INFO]    => URL: https://192.168.1.1 (verify SSL: false)
2021/08/18 15:21:58 [INFO]    => Version:  (2f7b6492-3e73-5c26-86d9-2d74e1587a42)
2021/08/18 15:21:58 [INFO]    => Username: mw25 (has password: true)
2021/08/18 15:21:58 [INFO]    => Hash PII / Poll Sites: false / all
2021/08/18 15:21:58 [INFO]    => Save Sites / Save DPI: true / false (metrics)
2021/08/18 15:21:58 [INFO]    => Save Events / Save IDS: false / false (logs)
2021/08/18 15:21:58 [INFO]    => Save Alarms / Anomalies: false / false (logs)
2021/08/18 15:21:58 [INFO]    => Save Rogue APs: false
2021/08/18 15:21:58 [INFO] Poller->InfluxDB started, interval: 30s, dp: false, db: unifi, url: http://127.0.0.1:8086
2021/08/18 15:21:58 [INFO] Loki config missing (or disabled), Loki output disabled!
2021/08/18 15:21:58 [INFO] Internal web server disabled!
2021/08/18 15:21:58 [INFO] Prometheus exported at http://0.0.0.0:9130/ - namespace: unifipoller
2021/08/18 15:22:35 [ERROR] influxdb.Write(points): {"code":"unauthorized","message":"Unauthorized"}
2021/08/18 15:22:59 [ERROR] influxdb.Write(points): {"code":"unauthorized","message":"Unauthorized"}
2021/08/18 15:23:30 [ERROR] influxdb.Write(points): {"code":"unauthorized","message":"Unauthorized"}
2021/08/18 15:23:59 [ERROR] influxdb.Write(points): {"code":"unauthorized","message":"Unauthorized"}
2021/08/18 15:24:30 [ERROR] influxdb.Write(points): {"code":"unauthorized","message":"Unauthorized"}
2021/08/18 15:24:59 [ERROR] influxdb.Write(points): {"code":"unauthorized","message":"Unauthorized"}
2021/08/18 15:25:31 [ERROR] influxdb.Write(points): {"code":"unauthorized","message":"Unauthorized"}
2021/08/18 15:25:59 [ERROR] influxdb.Write(points): {"code":"unauthorized","message":"Unauthorized"}
2021/08/18 15:26:29 [ERROR] influxdb.Write(points): {"code":"unauthorized","message":"Unauthorized"}
2021/08/18 15:27:01 [ERROR] influxdb.Write(points): {"code":"unauthorized","message":"Unauthorized"}
2021/08/18 15:27:29 [ERROR] influxdb.Write(points): {"code":"unauthorized","message":"Unauthorized"}
2021/08/18 15:28:13 [ERROR] influxdb.Write(points): {"code":"unauthorized","message":"Unauthorized"}

**Comment by davidnewhall** [comment]
Looks like your influx requires auth and you have the wrong credentials in your poller config file...

**Comment by mrax1** [comment]
On another note when I run 'brew services list' i get this output with errors for unpoller in Homebrew. This file looks like it is located in an unpoller folder as well. Does that need to be unif-poller as well? I set up unpoller with a command line inside Homebrew with code I got from the internet. 


Name      Status  User       Plist
emacs     started mw25 /Users/mw25/Library/LaunchAgents/homebrew.mxcl.emacs.plist
grafana   started mw25 /Users/mw25/Library/LaunchAgents/homebrew.mxcl.grafana.plist
influxdb   started mw25 /Users/mw25/Library/LaunchAgents/homebrew.mxcl.influxdb.plist
unbound  error   mw25 /usr/local/opt/unbound/homebrew.mxcl.unbound.plist
unpoller error   mw25 /usr/local/opt/unpoller/homebrew.mxcl.unpoller.plist

**Comment by mrax1** [comment]
David, how would I fix the influx DB issue. I didn't think I had credentials setup for the DB.

**Comment by mrax1** [comment]
I have installed and reinstalled influx db and it just won't work. I have tried to install version 1.8.9 because it says 2.0 will not function properly at unpoller.com.  I am also confused with what you said above about saving the config 
![Screen Shot 2021-08-19 at 1 52 05 PM](https://user-images.githubusercontent.com/86257730/130119452-fc9cd6ab-bfa1-4e61-9848-55551cd78d77.png)
file at /usr/local/etc/unifi-poller/up.conf. This allows unpoller to start up which I am happy about. But as I go back through the instructions and reinstall everything for the 20th time, it says to edit the config file at nano /usr/local/etc/unpoller/up.conf. So are those instructions wrong? I could have files everywhere with unpoller and unifi-poller. This is incredibly frustrating.

**Comment by davidnewhall** [comment]
The docs are ahead of the app. `unpoller` is not the name of any of that stuff yet. **The docs are wrong.** Once the app renames the docs will be accurate. Anywhere you see `unpoller`, look for `unifi-poller` too.

**Comment by mrax1** [comment]
> The docs are ahead of the app. `unpoller` is not the name of any of that stuff yet. **The docs are wrong.** Once the app renames the docs will be accurate. Anywhere you see `unpoller`, look for `unifi-poller` too.

Ok, thanks. Any thoughts on influx? It does have to be version 1.8.9 right? The docs say it can't be 2.0?

**Comment by davidnewhall** [comment]
Influx 2 changed everything and does not work out of the bix with poller. I've used 1.8.8 quite successfully, but I have not tried 1.8.9 yet. My thoughts are that it needs to work correctly and you need to be able to get into the database before poller is going to work. I do not have any special troubleshooting steps that you wont find in their docs.

**Comment by mrax1** [comment]
Good call. I reinstalled unpoller and will be working on getting influx install correctly. I downloaded version 1.8.4 for my intel MacBook Pro. But how do I get this installed in homebrew? All the other code lines I copy in the instructions download version 2.0 which obviously does not work. Just trying to get this installed in brew. Thanks

https://dl.influxdata.com/influxdb/releases/influxdb-1.8.4_linux_i386.tar.gz

**Comment by davidnewhall** [comment]
Looks like `brew install influxdb@1` should get you 1.8.9.

**Comment by mrax1** [comment]
Thanks, installed that several times but I keep getting a status error for influxdb.... searched the internet for days on this and just can't get it running. Almost ready to shut it down.

Name           Status   User       Plist
grafana        started  mw25 /Users/mw25/Library/LaunchAgents/homebrew.mxcl.grafana.plist
influxdb@1   error      mw25 /usr/local/opt/influxdb@1/homebrew.mxcl.influxdb@1.plist
unifi-poller   started  mw25 /Users/mw25/Library/LaunchAgents/homebrew.mxcl.unifi-poller.plist

**Comment by mrax1** [comment]
Got it up and running. Thank you for your insight.

**Comment by davidnewhall** [comment]
woohoo!
